project/MultiNode.scala

```diff
@@ -12,6 +12,11 @@ import sbt._
 import sbt.Keys._
 
 object MultiNode extends AutoPlugin {
+  
+  // MultiJvm tests can be excluded from normal test target an validatePullRequest
+  // with -Dakka.test.multi-in-test=false
```

Comment:
I'm adding this to CONTRIBUTING:

Also, tests tagged as `PerformanceTest`, `TimingTest`, `LongRunningTest` and all multi-node tests are excluded from PR validation.

You can exclude the same kind of tests in your local build by starting sbt with:

```
sbt -Dakka.test.tags.exclude=performance,timing,long-running -Dakka.test.multi-in-test=false
```